### PR TITLE
EOS-19185: remove links to internal sites.

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise <yashodhan.pise@seagate.com>

-----
Fixes issue with internal links:  
https://github.com/Seagate/cortx-prvsnr/search?p=1&q=satellite1

[View rendered Cortx-ProvisionerQuickstartGuide.md](https://github.com/ypise/cortx-prvsnr/blob/EOS-19185/Cortx-ProvisionerQuickstartGuide.md)
[View rendered files/scripts/profiling/README.md](https://github.com/ypise/cortx-prvsnr/blob/EOS-19185/files/scripts/profiling/README.md)

### CODE REVIEW

File: test/api/python/integration/test_api_inner.py [hunk 1]
```diff
@@ -635,7 +635,7 @@ def test_cortx_update():
     api_call(
         'set_swupdate_repo',
         '1.2.3',
-        source='http://cortx-storage.colo.seagate.com/releases/cortx/integration/centos-7.7.1908/last_successful',  # noqa: E501
+        source='http://<cortx_release_server>/releases/cortx/integration/centos-7.7.1908/last_successful',  # noqa: E501
```
Comment (ypise) [inline test/api/python/integration/test_api_inner.py:638]: @andkononykhin2 , @dmitrydb 
Need your attention here. What could be the best alternative?

File: test/cli/src/test_functions.py [hunk 2]
```diff
@@ -831,7 +831,7 @@ def test_functions_install_provisioner_rpm(
         'cat /etc/yum.repos.d/prvsnr.repo | grep baseurl'
     ).split('=')[1]
     assert baseurl == (
-        'http://cortx-storage.colo.seagate.com/releases/cortx/{}'
+        'http://<cortx_release_server>/releases/cortx/{}'
```
Comment (ypise) [inline test/cli/src/test_functions.py:834]: @andkononykhin2 , @dmitrydb 
Need your attention here. What could be the best alternative?

File: test/api/python/integration/test_api_inner.py (hunk 1 above)
Comment (andkononykhin2) [reply test/api/python/integration/test_api_inner.py:638]: These tests are outdated and are not run for the moment. So please just add a FIXME. Later we would need an input from @shailesh-vaidya: whether a release repo will be public or not. If not - we will make that thing optional so the test can be run inside Seagate network only.

File: test/cli/src/test_functions.py (hunk 2 above)
Comment (andkononykhin2) [reply test/cli/src/test_functions.py:834]: The same:  please just add FIXME note. We might not need that test anymore actually.

**Review by andkononykhin2 — APPROVED** [review]

**Pull request merged**